### DISCUSSION TITLE
The setting page will open when every boot, can I hide it?

### DISCUSSION THREAD

**Discussion by JasonLamv-t** [MonitorControl/MonitorControl#1220] [Q&A]
As the title.

**Comment by waydabber** [comment]
That simply should not happen. Can you post a screenshot of the setting page you are seeing?

**Reply by JasonLamv-t** [reply]
> That simply should not happen. Can you post a screenshot of the setting page you are seeing?

<img width="1586" alt="image" src="https://user-images.githubusercontent.com/39087996/202769204-aae7626b-fd56-40fc-84a9-3594aa6703ff.png">

It is the general page.

**Reply by waydabber** [reply]
Honestly, I have no idea why that happens. Super weird. 😕

Anybody else has this issue? Let me know!

**Reply by JasonLamv-t** [reply]
It doesn't big deal. I can close it manually.😃
